acsoo/templates/project/+project.name+/.pre-commit-config.yaml

```diff
@@ -1,12 +1,36 @@
 exclude: "^(?!odoo/)|.pot$|.po$"
+default_language_version:
+  python: python3
 repos:
-- repo: https://github.com/ambv/black
-  rev: 18.9b0
+- repo: https://github.com/psf/black
+  rev: 19.10b0
   hooks:
-  - id: black
-    language_version: python3
+    - id: black
+- repo: https://github.com/prettier/prettier
+  rev: 1.19.1
+  hooks:
+    - id: prettier
+# TODO Avoid awebdeveloper/pre-commit-prettier if possible
+# HACK https://github.com/prettier/prettier/issues/7407
```

Comment:
You can resolve this TODO with this: https://github.com/OCA/maintainer-quality-tools/pull/655/files#